tasks/install.yml

```diff
@@ -28,6 +28,15 @@
     state: present
   when: sansible_newrelic_integrations_infra_enabled
 
+- name: Ensure New Relic Infra log directory exists
+  become: yes
+  file:
+    group: "{{ sansible_newrelic_group }}"
+    owner: "{{ sansible_newrelic_user}}"
+    path: /var/log/newrelic
+    state: directory
+  when: sansible_newrelic_integrations_infra_enabled
```

Comment:
Can we please do

```yaml
when: sansible_newrelic_integrations_infra_enabled == true
```

?

Reply:
Sure, I've updated that in a few places, just waiting for the test re-run.